src/components/manage/UniversalLink/UniversalLink.jsx

```diff
@@ -24,9 +24,20 @@ const UniversalLink = ({
 
   let url = href;
   if (!href && item) {
-    url = flattenToAppURL(item['@id']);
-    if (!token && item.remoteUrl) {
-      url = item.remoteUrl;
+    if (!item['@id']) {
+      /* eslint no-console: 0 */
+      console.error(
+        'Invalid item passed to UniversalLink',
+        item,
+        props,
+        children,
+      );
+      url = '/';
```

Comment:
I would evaluate if setting ``url = '#';`` instead, in order to avoid rendering a link to the homepage with a different title.